docker/ray-ml/Dockerfile

```diff
@@ -5,16 +5,19 @@ FROM rayproject/ray:nightly"$GPU"
 COPY requirements.txt ./
 COPY requirements_ml_docker.txt ./
 COPY requirements_rllib.txt ./
-COPY requirements_tune.txt ./requirements_tune.txt
+# Docker image uses Python 3.7
+COPY linux-py3.7-requirements_tune.txt ./requirements_tune.txt
 
 RUN sudo apt-get update \
     && sudo apt-get install -y gcc \
         cmake \
         libgtk2.0-dev \
         zlib1g-dev \
         libgl1-mesa-dev \
-    && $HOME/anaconda3/bin/pip --use-deprecated=legacy-resolver --no-cache-dir install -r requirements.txt \ 
-    && $HOME/anaconda3/bin/pip --use-deprecated=legacy-resolver --no-cache-dir install -r requirements_ml_docker.txt \
+    && $HOME/anaconda3/bin/pip --use-deprecated=legacy-resolver --no-cache-dir install -r requirements.txt \
+    && $HOME/anaconda3/bin/pip --no-cache-dir install -r requirements_rllib.txt \
+    && $HOME/anaconda3/bin/pip --no-cache-dir install -r requirements_tune.txt \
+    && $HOME/anaconda3/bin/pip --no-cache-dir install -r -U requirements_ml_docker.txt \
```

Comment:
Is the docker ray-ml build tested?

Reply:
The Linux wheels test whether it can be built. Though I don't think there are any functionality tests that run in the docker containers.

Reply:
We do not actually test whether the built container works, but we do check that the container can build.